tsconfig.json

```diff
@@ -17,12 +17,7 @@
     "declaration": true,
     "downlevelIteration": true,
 
-    "suppressImplicitAnyIndexErrors": true,
-    "noImplicitAny": true,
-    "alwaysStrict": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "skipLibCheck": true,
```

Comment:
I am not sure if running type check for dependencies is a good idea. This might be overhead for our build process.

Reply:
It was an oversight by me. Thank you for spotting it.